readme.md

```diff
@@ -16,23 +16,23 @@ $ npm install --save file-icon
 const fs = require('fs');
 const fileIcon = require('file-icon');
 
-fileIcon('Safari').then(buffer => {
+fileIcon.buffer('Safari').then(buffer => {
 	fs.writeFileSync('safari-icon.png', buffer);
 });
 
-fileIcon('com.apple.Safari').then(buffer => {});
+fileIcon.buffer('com.apple.Safari', {size: 64}).then(buffer => {});
 
-fileIcon('unicorn.jpg').then(buffer => {});
+fileIcon.buffer('unicorn.jpg').then(buffer => {});
 
-fileIcon('Safari', {destination: 'safari-icon.png'}).then(() => {
+fileIcon.file('Safari', {destination: 'safari-icon.png'}).then(() => {
 	const buffer = fs.readFileSync('safari-icon.png');
```

Comment:
We already show how to get the buffer with `.buffer()`. Would make more sense to just show a `console.log('Done')` here or something.